test/pylib/log_browsing.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2023-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+from typing import Optional
+import pytest
+import os
+import socket
+
+from test.pylib.internal_types import IPAddress
+
+# Utility function for trying to find a local process which is listening to
+# the address and port to which our our CQL connection is connected. If such a
+# process exists, return its process id (as a string). Otherwise, return None.
+# Note that the local process needs to belong to the same user running this
+# test, or it cannot be found.
+def node_process_id(server_ip: IPAddress, port: int = 9042) -> Optional[int]: #TODO port
```

Comment:
> The word "local" in the original name I used for this function, "local_process_id", refers to the fact that it can only find a local process - it can't find a process that runs on some remote machine. I think you shouldn't have lost this word (or something with similar meaning).

Good point. I just wanted to distinguish these two functions. I will change that.

> I don't understand the comment "TODO port" - it seems port is already a parameter?

Sorry, It was a note for me and I deleted it in the later commit. I forgot to delete it from the first one.